src/App.tsx

```diff
@@ -1,24 +1,86 @@
 import React from 'react';
+import { Clock } from './components/Clock';
+
 import './App.scss';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isClockVisible: boolean;
+  clockName: number;
 };
 
-export default App;
+export class App extends React.Component<{}, State> {
+  state = {
+    isClockVisible: true,
+    clockName: Math.floor(Math.random() * 1000),
+  };
+
+  clockShow = () => {
```

Comment:
Назви методів мають починатися з дієслова
```suggestion
  showClock = () => {
```